elastic/kibana
# [Logs+] Add Filter Control Customization Point

closes https://github.com/elastic/kibana/issues/158561

## 📝  Summary

This PR adds a new customization point to allow for prepending custom filter controls to the search bar.
At the moment we are only showing a default namespace filter, once this is ready we will then check how to provide curated filters per integration.

## ✅  Testing

1. Make sure to have some documents to different data sets with different namespace, you can use [this document](https://www.elastic.co/guide/en/elasticsearch/reference/current/set-up-a-data-stream.html) as an example
2. Navigate to Discover with the log-explorer profile, `/p/log-explorer`
3. Validate that the new filter control is there
4. Filter using this new control and make sure documents are filtered out

## 🎥 Demo

https://github.com/elastic/kibana/assets/11225826/6828f62f-dd09-42bd-930c-dd7eaf94958b
